carapp/db_models.py

```diff
@@ -1,46 +1,52 @@
 import datetime
+from sqlalchemy.orm import relationship
 from carapp import db
 
 
 class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
PK should be auto-incremented if no additional requirements present. 👍🏼

Reply:
💯